x-pack/legacy/plugins/siem/public/components/page/network/users_table/index.tsx

```diff
@@ -95,7 +115,7 @@ const UsersTableComponent = React.memo<UsersTableProps>(
           }
         }
       },
-      [sort, type]
+      [sort, type, updateNetworkTable]
```

Comment:
To match the other tables, missing `tableType` here:

```suggestion
      [sort, type, tableType, updateNetworkTable]
```